src/OrderBookHandlers.ts

```diff
@@ -50,7 +50,7 @@ OrderBookContract.OrderChangeEvent.handler(({ event, context }) => {
       timestamp,
     }
     : null;
-  const idSource = `${event.transactionId}-${timestamp}-${event.data.order_id}`;
+  const idSource = `${event.transactionId}-${timestamp}-${event.data.order_id}-${event.receiptIndex}`;
```

Comment:
Add receipt index to `idSource` to prevent events in same transaction from getting confused